src/main/java/seedu/address/commons/core/GuiSettings.java

```diff
@@ -1,6 +1,6 @@
 package seedu.address.commons.core;
 
-import java.awt.Point;
+import java.awt.*;
```

Comment:
Import statements should be listed out fully and individually instead of using the * modifier to import the entire package. Please refer to the following textbook portion. https://nus-cs2103-ay1819s2.github.io/cs2103-website/coding-standards/java/basic.html#package-and-import-statements